edx_notifications/server/api/api_utils.py

```diff
@@ -44,7 +44,7 @@ def authenticate(self, request):
         except User.DoesNotExist:
             return None
 
-        if not token == settings.NODEBB_MASTER_TOKEN or not user.is_active:
+        if not token == settings.NODEBB_MASTER_TOKEN:
```

Comment:
@RanaWaqar7550 is it possible to use `token is not settings.NODEBB_MASTER_TOKEN` or `token != settings.NODEBB_MASTER_TOKEN`